server/app.js

```diff
@@ -42,7 +40,7 @@ if (process.env.NODE_ENV === "development") {
   app.use(logger("dev"));
 }
 
-app.use(cors());
+// app.use(cors());
```

Comment:
Is there an issue with CORS right now? How could this be addressed in either this PR or a future PR?

Reply:
There is no issue with CORS, but it was originally added to find a solution for the problems we were facing with google authentication. We found that the source of the problem was database related, but communication between the front end and the back end. Therefore, it was something that was added that was not needed in our code, and we decided to remove it.

Reply:
Okay